.github/workflows/packaging.yml

```diff
@@ -3,55 +3,166 @@ name: Building Cobbler packages
+        uses: actions/upload-artifact@v3
+        with:
+          name: rpms-opensuse-leap
+          path: |
+            rpm-build/*.rpm
   build-opensuse-tumbleweed-rpms:
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Build a openSUSE Tumbleweed Package
         shell: 'script -q -e -c "bash {0}"'
         run: |
           ./docker/rpms/build-and-install-rpms.sh opensuse-tumbleweed docker/rpms/opensuse_tumbleweed/openSUSE_TW.dockerfile
+      - name: Rename RPM
+        run: |
+          file-rename -v -d -e 's/koan-(\d+\.\d+\.\d+)-1\.(\w+)\.rpm/koan-$1-1.tw.$2.rpm/' rpm-build/*.rpm
+      - name: Archive RPMs
+        uses: actions/upload-artifact@v3
+        with:
+          name: rpms-opensuse-tumbleweed
+          path: |
+            rpm-build/*.rpm
   build-debian10-debs:
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Build a Debian 10 Package
         shell: 'script -q -e -c "bash {0}"'
         run: |
           ./docker/debs/build-and-install-debs.sh deb10 docker/debs/Debian_10/Debian10.dockerfile
   build-debian11-debs:
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Build a Debian 11 Package
         shell: 'script -q -e -c "bash {0}"'
         run: |
           ./docker/debs/build-and-install-debs.sh deb11 docker/debs/Debian_11/Debian11.dockerfile
+      - name: Archive DEBs
+        uses: actions/upload-artifact@v3
+        with:
+          name: debs-debian
+          path: |
+            deb-build/DEBS/all/*.deb
+  build-wheel:
+    name: Build Python Wheel
+    runs-on: ubuntu-latest
+    # TODO: Move directly to Ubuntu - libvirt problems on GHA
+    container: registry.opensuse.org/opensuse/tumbleweed:latest
+    steps:
+      - name: Install system dependencies
+        run: >-
+          zypper -n in --no-recommends
+          git
+          tar
+          make
+          python3
+          python3-base
+          python3-wheel
+          python3-build
+          python3-setuptools
+          python3-pip
+          python3-libvirt-python
+          python3-Sphinx
+          python3-netifaces
+          tree
+      - uses: actions/checkout@v3
+      - name: Mark directory as safe for Git
+        run: git config --global --add safe.directory /__w/koan/koan
+      - name: Install dependencies
+        run: pip3 install .
+      - name: Build a binary wheel and a source tarball
+        run: make release
+      - name: Show tree
+        run: tree
+      - name: Archive Wheel
+        uses: actions/upload-artifact@v3
+        with:
+          name: wheel
+          path: |
+            dist/*.whl
+            dist/*.tar.gz
+  create-release:
+    name: Build the release and create a GitHub release
+    runs-on: ubuntu-latest
+    needs: [
+      build-rockylinux-rpms,
+      build-fedora-rpms,
```

Comment:
The workflows are actually named `build-rockylinux8-rpms` and `build-fedora37-rpms`. Imo we should name the workflows version agnostic but if the version is needed I would at least suggest renaming the Fedora one to the newest supported version

Reply:
Actually not only that but I also forgot to include the Ubuntu DEBs.